src/test/java/com/qunite/api/web/QueueControllerIntegrationTest.java

```diff
@@ -67,6 +65,7 @@ public void cleanAll() {
   }
 
   @Test
+  @WithMockUser("First")
```

Comment:
exctract "First" to field? Same for all `@WithMockUser`

Reply:
User may vary depending on test logic. And i dont think that putting a variable in annotation is a good idea.